test/Api.Tests/TestStartup.cs

```diff
@@ -14,7 +12,9 @@ public void ConfigureTestServices(IServiceCollection services)
             base.ConfigureServices(services);
 
             // Register here the dependency injection services
-           // services.AddDbContext<ApplicationDbContext>(opts => opts.UseInMemoryDatabase());
```

Comment:
I tried with an own start up class but there was a framework level issue or some problem which I was not able to figure it out. Later contacted the Admin of the framework and hence used the same as per his example. And as per the framework, we need not configure explicitly the In-memory database. The framework internally uses In-memory database whenever needed. Please check out this tutorial :  https://github.com/ivaylokenov/MyTested.AspNetCore.Mvc/tree/master/samples/MusicStore/MusicStore.Test